CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ Entries in this file should be limited to:
 - Obscure side-effects that are not obviously apparent based on the JIRA associated with the changes.
 Please avoid adding duplicate information across this changelog and JIRA/doc input pages.
 
+## [NEXT RELEASE]
+
+### Added Features
+
+- ROX-25066: Add new external backup integration for non-AWS S3 compatible providers.
+
+### Removed Features
+
+### Deprecated Features
+
+### Technical Changes
+
+
```

Comment:
```suggestion
```

Was ROX-25066 cherry-picked to the release branch for 4.5.1?
If yes, rename NEXT RELEASE to 4.5.1.

If not, please remove the entire section.

Reply:
Dropped this section because this note is for the next release and not patch